xep-0369.xml

```diff
@@ -279,7 +282,7 @@
         <li>'Date of Configuration Change'.  Encoded as the item ID.</li>
         <li>'Last Change Made By'.  Bare JID of the user making the last change.</li>
         <li>'Owners'.  List of bare JIDs with Owner rights as defined in ACL node.  When a list is created, the JID creating the list is configured as an owner, unless this attribute is explicitly configured to another value.  Owners will generally have full rights to make changes to channel configuration and access control.</li>
-        <li>'Administrators'.  List of bare JIDs with Administrator rights.  An Administrator has rights to make changes to channel access control and configuration as defined in ACL node.  An Administrator also has operational right on a channel, such a kicking users out of the channel.  These rights are analogous to moderator rights defined in &xep0045;.</li>
+        <li>'Administrators'.  List of bare JIDs with Administrator rights.  An Administrator has rights to make changes to channel access control and configuration as defined in ACL node.  An Administrator also has operational right on a channel, such a kicking users out of the channel. These rights are analogous to moderator rights defined in <cite>Multi-User Chat</cite>.</li>
```

Comment:
I'm not sure how we generally do this; there are a ton of citations for 0045 in here though, and I feel like there should only be 1 and the rest should just link to that or just be formatting. This might be a discussion for the editors / a future amendment to the XEP style guide.